<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AptOfflineMain</class>
 <widget class="QMainWindow" name="AptOfflineMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>432</width>
    <height>544</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>432</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>APT Offline</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createProfileButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>371</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Generate Signature</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/contact-new.png</normaloff>:/icons/icons/contact-new.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Download Packages or Updates</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/go-down.png</normaloff>:/icons/icons/go-down.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="restoreButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Install Packages or Updates</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/install.png</normaloff>:/icons/icons/install.png</iconset>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>371</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <widget class="QLabel" name="descriptionField">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>371</width>
       <height>211</height>
      </rect>
     </property>
     <property name="text">
      <string>Hover your mouse over the buttons to get the description</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>10</number>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/icons/icons/application-exit.png</normaloff>:/icons/icons/application-exit.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOperations">
    <property name="title">
     <string>Operations</string>
    </property>
    <addaction name="menuCreateProfile"/>
    <addaction name="menuDownload"/>
    <addaction name="menuInstall"/>
    <addaction name="separator"/>
    <addaction name="menuExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="menuHelp_2"/>
    <addaction name="menuAbout"/>
   </widget>
   <addaction name="menuOperations"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="menuCreateProfile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/contact-new.png</normaloff>:/icons/icons/contact-new.png</iconset>
   </property>
   <property name="text">
    <string>Generate Signature</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="menuDownload">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/go-down.png</normaloff>:/icons/icons/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Download Packages or Updates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="menuInstall">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/install.png</normaloff>:/icons/icons/install.png</iconset>
   </property>
   <property name="text">
    <string>Install Packages or Updates</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="menuExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/application-exit.png</normaloff>:/icons/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="menuHelp_2">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/help-contents.png</normaloff>:/icons/icons/help-contents.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="menuAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/help-about.png</normaloff>:/icons/icons/help-about.png</iconset>
   </property>
   <property name="text">
    <string>About apt-offline</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
